### ISSUE TITLE
RomBanks 0x7 overflowing Byte?

### ISSUE THREAD

**Issue by maxbit89** [drhelius/arduinogameboy#2]
Hy i looked through the Code and i noticed that in case the ROM Size equals 0x7 you set romBanks to 256 (0x100), since romBanks is from type byte this would overflow.

**Comment by drhelius** [comment]
Nice catch you are completely right, this code is from a looong time ago.

Anyway I don't think there is a cartridge that big (256 Mbit) apart from may be any kind of modern homebrew. I think almost all are under 64Mbit.

**Comment by maxbit89** [comment]
Yeah i wasn't even thinking some one would look at this ^^. Just wanted to document in case some one ran into that problem.
